README.md

```diff
@@ -62,13 +63,16 @@ You can then use the following selector anywhere in your project:
 | borderRadius | string | ```34px``` | Sets the track border radius |
 
 ### events
-| Event Name | Returns | Description |
```

Comment:
I prefer the table structure - it is much more friendly and easy to understand.
- You can write the 3 main events like change, input and focus 
- and below the table add an asterisk like so:
* Other then the mentioned events the component supports all standard HTML `<input>` events